metis-core/metis-core-rest/src/main/java/eu/europeana/metis/core/rest/config/OrchestratorConfig.java

```diff
@@ -238,13 +254,30 @@ public SchedulerExecutor getSchedulingExecutor(OrchestratorService orchestratorS
         redissonClient);
     return schedulerExecutor;
   }
+  
+  @Bean
+  public QueueConsumer getQueueConsumer(WorkflowExecutorManager workflowExecutionManager,
+      WorkflowExecutionMonitor workflowExecutionMonitor,
+      @Qualifier("rabbitmqConsumerChannel") Channel rabbitmqConsumerChannel) throws IOException {
+    return new QueueConsumer(rabbitmqConsumerChannel, propertiesHolder.getRabbitmqQueueName(),
+        workflowExecutionManager, workflowExecutionManager, workflowExecutionMonitor);
+  }
+
+  @Scheduled(fixedDelayString = "${periodic.execution.monitor.check.in.millisecs}",
+      initialDelayString = "${periodic.execution.monitor.check.in.millisecs}")
+  public void runWorkflowExecutionMonitor() {
```

Comment:
Why having this running since we do the exact same thing during the failsafe? I mean having inactive RUNNING executions is quite rare to the application, so I feel like this will most of the time be a waste of resources, what do you think?

Reply:
We discussed this and agreed. It will be removed.